submissions/lalalilulu/tiny-world-pre-oop/index.js

```diff
@@ -0,0 +1,77 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/lalalilulu/a-tiny-JS-world
+   Web app: https://lalalilulu.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+
+    constructor(species, name, gender, saying, friends = []) {
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.saying = saying;
+        this.friends = Array.from(friends).join(", ");
```

Comment:
What is the purpose of `.from` here?

Reply:
Array.from() method was a possible way to handle a type error, if a single value is passed for the field ‘friends’ (not an object array).  

Uncaught TypeError: friends.join is not a function
    at new Inhabitant (index.js:19)
    at new Animal (index.js:30)
    at new Cat (index.js:57)
    at index.js:71